docs/data_engineers/accessing_data/elt_pipeline/elt_faq.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 4
+---
+
+# ELT FAQ
```

Comment:
@cgkournelos-rf Let's name this just "FAQ` since the ELT keyword is already mentioned in top sidebar category.

Reply:
fixed